mongodbatlas/data_source_mongodbatlas_cloud_provider_snapshot_restore_jobs_test.go

```diff
@@ -36,7 +36,7 @@ func TestAccDataSourceMongoDBAtlasCloudProviderSnapshotRestoreJobs_basic(t *test
 	})
 }
 
-func testAccMongoDBAtlasCloudProviderSnapshotRestoreJobsConfig(projectID, clusterName, description, retentionInDays string) string {
+func testAccMongoDBAtlasCloudProviderSnapshotRestoreJobsConfig(projectID, clusterName, description, retentionInDays string, pageNum, itemPage int) string {
 	return fmt.Sprintf(`
```

Comment:
Why did you not create another test case to keep the basic test case without those new parameters? I mean I think it is better to create another test case with the new feature instead of updating the existing test case.

Reply:
You're right, I'll do that, thank you very much, paco